### PR TITLE
Tweak clang-format

### PR DESCRIPTION
**Pull request by lucasdemarchi**
A few formatting and struggled with while doing some refactors in #143. First commits are actually from that PR and can be ignored here, I'm only including them because otherwise we would have some needless conflicts.

### CODE REVIEW

File: .clang-format [hunk 1]
```diff
@@ -20,6 +20,11 @@ AllowShortFunctionsOnASingleLine: Inline
 AlignConsecutiveDeclarations: false
 AlignConsecutiveAssignments: false
 AlignConsecutiveMacros: Consecutive
+
+BinPackArguments: false
+AllowAllParametersOfDeclarationOnNextLine: false
+AllowAllArgumentsOnNextLine: false
```
Comment (jbeyerstedt) [inline .clang-format:26]: I think, I also fiddled around with these parameters in my previous commits, because the arguments list inconsistencies bothered me. But didn't want to change too much about the format rules back then.
tl;dr: This is fine.

**Review by jbeyerstedt — APPROVED** [review]
